cmake/SundialsBuildOptionsPre.cmake

```diff
@@ -311,26 +311,85 @@ endif()
 # Options for SUNDIALS testing
 # ---------------------------------------------------------------
 
+sundials_option(SUNDIALS_TEST_ENABLE_DEV_TESTS BOOL "Include development tests"
+                OFF ADVANCED)
+
+sundials_option(SUNDIALS_TEST_ENABLE_UNIT_TESTS BOOL "Include unit tests" OFF
```

Comment:
This would be a breaking option name change, yes?

Reply:
I've added the old options to `SundialsDeprecated.cmake`